app/helpers/application_helper.rb

```diff
@@ -14,7 +14,11 @@ def github_client
   def admin?
     @user.role.eql?("admin")
   end
-  
+
+  def staff?
```

Comment:
I think this method should probably live in the `user` model so you can do `user.staff?` or `user.admin?`